test/client_ui_test.py

```diff
@@ -84,6 +85,74 @@ def test_draw():
     assert f.last == line.ljust(79) + "\n"
 
 
+def test_json_format():
```

Comment:
The old test is called test_draw - we need to change both, maybe:
- test_draw_text
- test_draw_json